flake.nix

```diff
@@ -15,11 +15,11 @@
       inputs.nixpkgs.follows = "nixpkgs";
     };
     boba-backend = {
-      url = "github:bobaboard/boba-backend/devenv";
+      url = "github:bobaboard/boba-backend/more-nix";
       inputs.nixpkgs.follows = "nixpkgs";
     };
     boba-frontend = {
-      url = "github:bobaboard/boba-frontend/nix-flake";
+      url = "github:bobaboard/boba-frontend/main";
```

Comment:
The default branch is used automatically, we can set this input to just be "github:bobaboard/boba-frontend"